assets/templates/partials/banners/census.tmpl

```diff
@@ -5,11 +5,19 @@
                 {{ if eq .Language "en"  }}
                     <a class="flex" alt="Census 2021" href="/census"><img src="https://cdn.ons.gov.uk/assets/images/census-logo/logo-census-2021-white-landscape.svg" title="Census 2021" class="header__svg-logo margin-right--1" xmlns="http://www.w3.org/2000/svg"
                     focusable="false" width="167" height="32" viewBox="0 0 242 44"/>
-                    <a class="margin-top--0 text--white font-size--18 underline-link" href="/census">Find out more about census</a>
+                    {{if .FeatureFlags.EnableCensusResults}}
+                        <a class="margin-top--0 text--white font-size--18 underline-link" href="/census">Data and analysis from Census 2021</a>
+                    {{else}}
+                        <a class="margin-top--0 text--white font-size--18 underline-link" href="/census">Find out more about census</a>
+                    {{end}}
                 {{ else }}
```

Comment:
Needs to be consistent with the spacing in the templating in the rest of the file.

Reply:
sorted!

Reply:
And also the within the curly brackets. `{{ if .FeatureFlags.EnableCensusResults }} {{ else }} {{ end }}`